src/protocol/ipa/mod.rs

```diff
@@ -233,6 +233,111 @@ where
     .await
 }
 
+/// # Errors
+/// Propagates errors from multiplications
+/// # Panics
+/// Propagates errors from multiplications
+#[allow(dead_code)]
+pub async fn ipa_malicious_wip<F, B>(
+    sh_ctx: SemiHonestContext<'_, F>,
+    input_rows: &[IPAInputRow<F, B>],
```

Comment:
I think this is now a bit out of date. When I git pull, I see it as:
```
input_rows: &[IPAInputRow<F, MK, BK>],
```

Reply:
MK = BitArray trait for match keys
BK = BitArray trait for breakdown keys

@richajaindce ping me if you need help with the new code.